UI/output-obj.hpp

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include "obs.hpp"
+
+enum class OutputType {
```

Comment:
It may be smart to namespace this to prevent enum values leaking into the main namespace of any files that include the header.